tests/disk_management/test_disk_read.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+import unittest
+import os
+import egg.disk_management
+
+from tests.disk_management import load_expected_results, VHD_DIR
+
+
+@unittest.skipUnless(os.getuid() == 0, None)
+class TestDiskRead(unittest.TestCase):
+
+    scenario_data: dict
+    vhd_name: str
+    vhd_table: str
+    loop_path: str
+
+    @classmethod
+    def fillClassVariables(cls, scenario_data):
+        cls.vhd_name = scenario_data.get('vhd_name')
+        cls.vhd_table = VHD_DIR + cls.vhd_name + '/vhd.table'
```

Comment:
String formatting can be achieved using f-strings (if you have python 3.6 at least), like `f"{VHD_DIR}/{cls.vhd_name}/vhd.table"`